doc/release_notes.rst

```diff
@@ -11,6 +11,8 @@ Release Notes
 Upcoming Release
 ================
 
+* Added `investment` column to all components. These are the total upfront investments. The column is intended only for reporting and has no effect on the optimisation. For offshore wind, the `investment` column contains only investments into the wind park, the `connection_investment` is reported in a separate column.
```

Comment:
Can you add this also to the `doc/configtables/costs.csv` that the investment column has no effect on optimisation?